examples/rln-js/src/services/waku.ts

```diff
@@ -84,12 +85,15 @@ export class Waku implements IWaku {
     this.initEncoder(options);
     this.decoder = new RLNDecoder(
       options.rln.rlnInstance,
-      createDecoder(this.contentTopic)
+      createDecoder(this.contentTopic, { clusterId: 1 })
     );
 
     if (!this.node) {
       this.emitStatusEvent(WakuStatusEventPayload.INITIALIZING);
-      this.node = await createLightNode({ defaultBootstrap: true });
+      this.node = await createLightNode({
+        defaultBootstrap: true,
+        shardInfo: { clusterId: 1, contentTopics: [CONTENT_TOPIC] },
```

Comment:
This seems like a problem: if `this.contentTopic` changes then node will not be able to work. Let's remove dynamic setting of `contentTopic` form the app.